src/pages/dar_application/ResearcherInfo_new.js

```diff
@@ -0,0 +1,641 @@
+            onChange: ({key: name, value}) => {
+              const formattedValue = isNil(value) ? '' : formatSOString(value.displayName, value.email);
+              formFieldChange({name, formattedValue});
+            },
+            selectOptions: allSigningOfficials,
+            creatableConfig: {
+              placeholder: 'Signing Official',
+              getOptionLabel: (option) => formatSOString(option.displayName, option.email), //formats labels on dropdown
+              getNewOptionData: (inputValue) => { //formats user input into object for use within Creatable
+                return { displayName: inputValue };
+              },
+              getOptionValue: (option) => { //value formatter for options, attr used to ensure empty strings are treated as undefined
+                if(isNil(option) || isEmpty(option.displayName)) {
+                  return null;
+                }
+                return option;
+              }
+            },
+          }),
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h(FormField, {
+            id: 'itDirector',
+            description: 'I certify that the individual listed below is my IT Director',
+            placeholder: 'IT Director',
+            title: '1.7 Information Technology (IT) Director',
+            validators: [FormValidators.REQUIRED],
+            ariaLevel: ariaLevel + 1,
+            onChange: ({key: name, value,}) => formFieldChange(name, value),
+            defaultValue: itDirector
+          })
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          h3(['1.8 Cloud Use Statement*']),
+          span([
+            'Will you perform all of your data storage and analysis for this project on the ',
+            a({
+              rel: 'noopener noreferrer',
+              href: 'https://anvil.terra.bio/',
+              target: '_blank'
+            }, ['AnVIL']),
+            '?'
+          ]),
+          div([cloudRadioGroup])
+        ]),
+
+        div({className: 'dar-application-row'}, [
+          div({className: 'row no-margin'}, [
+            div({
+              isRendered: !anvilUse && anvilUse !== '',
+              className: 'computing-use-container',
+              style: {
+                backgroundColor: showValidationMessages && isCloudUseInvalid ? 'rgba(243, 73, 73, 0.19)' : 'inherit'
+              }
+            }, [
+              div({className: 'row no-margin'}, [
+                div({className: 'col-lg-12 col-md-12 col-sm-12 col-xs-12 rp-group'}, [
+                  input({
+                    type: 'checkbox',
+                    id: 'cloud-requested',
+                    name: 'cloudUse',
+                    className: 'checkbox-inline rp-checkbox',
+                    disabled: !isNil(darCode),
+                    required: true,
+                    checked: cloudUse,
+                    onChange: (e) => formFieldChange({name: 'cloudUse', value: e.target.checked})
```

Comment:
use formfield?

Reply:
Updated both of these inputs to formfield checkboxes!